src/main/java/core/basesyntax/model/zoo/Animal.java

```diff
@@ -1,9 +1,33 @@
 package core.basesyntax.model.zoo;
 
+import javax.persistence.DiscriminatorColumn;
+import javax.persistence.DiscriminatorType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+
+@Entity
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
+@DiscriminatorColumn(name = "animal",
+        discriminatorType = DiscriminatorType.STRING)
 public class Animal {
```

Comment:
You missed annotation @Table(name = "animals") that would be right naming for tables. same for other classes